order.js

```diff
@@ -280,3 +339,150 @@ Render.proof = (proof) => {
     </div>
   `;
 }
+function ticketItemString(data) {
+  var string = '';
```

Comment:
same as
```
var string = `${data.custFirst} ${data.custLast}, ${data.email}, ${data.orderId_raw}`;
```

Reply:
oh, this is never called by anyone? Don't check in logic that's not used.
Before you commit you should be looking at all the diffs of new stuff and erasing anything extra.